assets/prometheus-k8s/rules.yaml

```diff
@@ -349,6 +349,30 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container_name!="POD",container_name!="",pod_name!=""}[5m]))
         / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: kube_node_labels and on(node) kube_node_spec_taint{key="node-role.kubernetes.io/master"}
+      labels:
+        label_node_role_kubernetes_io: master
+      record: cluster:master_nodes
+    - expr: sum((cluster:master_nodes * on(node) group_left kube_node_status_capacity_cpu_cores)
+        or on(node) (kube_node_labels * on(node) group_left kube_node_status_capacity_cpu_cores))
+        BY (label_beta_kubernetes_io_instance_type, label_node_role_kubernetes_io)
+      record: cluster:capacity_cpu_cores:sum
+    - expr: sum((cluster:master_nodes * on(node) group_left kube_node_status_capacity_memory_bytes)
+        or on(node) (kube_node_labels * on(node) group_left kube_node_status_capacity_memory_bytes))
+        BY (label_beta_kubernetes_io_instance_type, label_node_role_kubernetes_io)
+      record: cluster:capacity_memory_bytes:sum
+    - expr: sum(1 - rate(node_cpu_seconds_total{mode="idle"}[1m]) * on(namespace,
+        pod) group_left(node) node_namespace_pod:kube_pod_info:{})
+      record: cluster:cpu_usage_cores:sum
+    - expr: sum(node:node_memory_bytes_total:sum - node:node_memory_bytes_available:sum)
+      record: cluster:memory_usage_bytes:sum
+    - expr: sum(rate(container_cpu_usage_seconds_total{namespace=~"openshift-.+"}[1m]))
+      record: cluster:infra_cpu_usage_cores:sum
+    - expr: sum(container_memory_working_set_bytes{namespace=~"openshift-.+"})
+      record: cluster:infra_memory_usage_bytes:sum
```

Comment:
same as above regarding openshift prefix as opposed to cluster

Reply:
ack